src/screencast/wlr_screencast.c

```diff
@@ -543,7 +543,11 @@ struct xdpw_wlr_output *xdpw_wlr_output_find(struct xdpw_screencast_context *ctx
 }
 
 static void wlr_remove_output(struct xdpw_wlr_output *out) {
+	free(out->name);
+	free(out->make);
+	free(out->model);
 	wl_list_remove(&out->link);
+	free(out);
 }
```

Comment:
Should out->xdg_output also be freed here? It's returned from [zxdg_output_manager_v1_get_xdg_output](https://github.com/emersion/xdg-desktop-portal-wlr/blob/master/protocols/xdg-output-unstable-v1.xml#L71).

Reply:
Yes, via `zxdg_output_v1_destroy`.

Reply:
We should also free the `wl_output`, via `wl_output_destroy`.

Reply:
thanks. added it.